network/devices/link.go

```diff
@@ -78,6 +99,33 @@ func (Link) List() (names []string, err error) {
 	return names, nil
 }
 
+func (l Link) GetStatistics() (garden.ContainerNetworkStat, error) {
```

Comment:
it's probably slightly more idiomatic Go to have the method be called `Statistics` rather than `GetStatistics`